code/modules/mob/living/damage_procs.dm

```diff
@@ -73,7 +73,6 @@
 		if(JITTER)
 			if(status_flags & CANSTUN)
 				jitteriness = max(jitteriness,(effect * blocked))
-	updatehealth()

```

Comment:
not needed. If an update is required, Weaken(), Paralyse() or Stun() will call it.